qCC/ccGraphicalTransformationTool.cpp

```diff
@@ -508,11 +539,13 @@ void ccGraphicalTransformationTool::advTranslateRefUpdate(int index)
 
 void ccGraphicalTransformationTool::advRotateComboBoxUpdate(int index)
 {
+	rotComboBoxItems selectedAxis = (rotComboBoxItems)rotComboBox->itemData(index).toInt();
```

Comment:
```suggestion
	rotComboBoxItems selectedAxis = static_cast<rotComboBoxItems>(rotComboBox->itemData(index).toInt());
```

Reply:
Yep, but this "old fashioned" type of cast was uses elsewhere in the code so in the end I stuck to this style. It's now modified everywhere (a "proper" static_cast) in the next commit.